eclair-core/src/main/resources/reference.conf

```diff
@@ -37,14 +37,12 @@ eclair {
   node-color = "49daaa"
 
   trampoline-payments-enable = false // TODO: @t-bast: once spec-ed this should use a global feature flag
-  global-features = "0200" // variable_length_onion
-  local-features = "088a" // initial_routing_sync + option_data_loss_protect + option_channel_range_queries + option_channel_range_queries_ex
+  features = "0a8a" // initial_routing_sync + option_data_loss_protect + option_channel_range_queries + option_channel_range_queries_ex + variable_length_onion
```

Comment:
I wonder if we should use a completely different method, using `rfcName` field:

```
features {
  option_data_loss_protect = "mandatory"
  initial_routing_sync = "optional"
  gossip_queries = "mandatory"
  ...
} 
```

Less error prone and more readable, but also less flexible.

Reply:
Would also affect the `override-features` attribute...

Reply:
I think it's a good idea if we really want to allow users to turn some features on and off.
The problem is there are quite a few features that we can't really turn off right now (some code paths will ignore the fact that a feature is turned off and still use it).
I think it makes sense to do that change once we've worked on making sure the code base really allows completely turning off all features.